src/test/java/com/deloitte/ddwatch/services/ProjectServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package com.deloitte.ddwatch.services;
+
+
+import com.deloitte.ddwatch.model.Project;
+import com.deloitte.ddwatch.model.ProjectRepo;
+import com.deloitte.ddwatch.repositories.ProjectRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ProjectServiceTest {
+
+    @InjectMocks
+    ProjectService projectService;
+
+    @Mock
+    private ProjectRepository projectRepository;
+
+    @Mock
+    private TagService tagService;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void createProject() {
```

Comment:
Test method names usually follow the pattern `<method_name>Should<action>When<optional_condition>`. For this case, it could be something like: **createProjectShouldReturnProjectWithNoTags**. It's long but it documents what the test verifies. Rename all accordingly.